script.js

```diff
@@ -0,0 +1,118 @@
+//     Task 1
+
+let str = "true";
+let boo = false;
+let num = 17;
+let und = undefined;
+let nul = null;
+
+console.log(typeof str, typeof boo, typeof num, typeof und, typeof nul);
+
+//     Task 2
+
+let height = 15;
+let width = 20;
+
+if (height > width) {
+  console.log(height);
+} else {
+  console.log(width);
+}
+
+//     Task 3
+
+for (let i = 1; i <= 20; i++) {
+  if (i % 3 === 0) {
+    console.log(i);
+  }
+}
+
+//     Task 4
+
+let key = true;
+let documents = true;
+let pen = true;
+let apple = false;
+let orange = true;
+
+let shouldGoToWork = key && documents && pen && (apple || orange);
+console.log(shouldGoToWork);
+
+//     Task 5
+
+let numFB = prompt("Введите число");
+
+if (numFB % 5 === 0) {
+  console.log("Fiz");
+} else if (numFB % 3 === 0) {
+  console.log("Buz");
+} else if (numFB % 5 === 0 && numFB % 3 === 0) {
+  console.log("FizBuz");
+}
```

Comment:
It is a good practice to add ELSE to the end of your condition.

Reply:
If only one "if", does it have to be "else"?

Reply:
No, you have to look depending on the conditions and tasks. In a case, with a prompt, we can enter data in which no IF will work, so in this situation it is important to have ELSE